src/rpc/util.cpp

```diff
@@ -699,6 +699,15 @@ std::pair<int64_t, int64_t> ParseDescriptorRange(const UniValue& value)
     return {low, high};
 }
 
+int64_t ParseInteger(const std::string& param_name, const UniValue& value)
```

Comment:
This function isn't necessary. `get_int64()` will throw an error if the `UniValue` object is not an number.

Reply:
Yeah so I was trying to get a more specific error message here, to be consistent with the rest of the input validation (instead of having `get_int64()` throw a generic error). Maybe overkill?

Reply:
I think it's a overkill to do that.

Reply:
Kk will remove.